client-src/elements/chromedash-banner.ts

```diff
@@ -1,19 +1,13 @@
 import {LitElement, css, html, nothing} from 'lit';
 import {SHARED_STYLES} from '../css/shared-css.js';
+import {customElement, property} from 'lit/decorators.js';
 
+@customElement('chromedash-banner')
 class ChromedashBanner extends LitElement {
-  static get properties() {
-    return {
-      timestamp: {type: Number},
-      message: {type: String},
-    };
-  }
-
-  constructor() {
-    super();
-    this.timestamp = null; // Unix timestamp: seconds since 1970-01-01.
-    this.message = '';
-  }
+  @property({type: Number})
+  timestamp: null | number = null; // Unix timestamp: seconds since 1970-01-01.
```

Comment:
While you're moving things anyway, it'd be good to use the jsdoc format for this comment:
```suggestion
  /** Unix timestamp: seconds since 1970-01-01. */
  timestamp: null | number = null;
```